common-money.lic

```diff
@@ -28,8 +30,16 @@ module DRCM
     amount
   end
 
+  def hometown_currency(hometown_name)
+    return currency = get_data('town')[hometown_name]['currency']
```

Comment:
You don't need to assign to `currency` here (or even use an explicit `return`)

Reply:
Updated